frontend/app/tests/unit/specs/components/history/events/eth-block-event-form.spec.ts

```diff
@@ -0,0 +1,206 @@
+import {
+  type ThisTypedMountOptions,
+  type Wrapper,
+  mount
+} from '@vue/test-utils';
+import { createPinia, setActivePinia } from 'pinia';
+import Vuetify from 'vuetify';
+import { HistoryEventEntryType } from '@rotki/common/lib/history/events';
+import { type EthBlockEvent } from '@/types/history/events';
+import EthBlockEventForm from '@/components/history/events/forms/EthBlockEventForm.vue';
+
+describe('EthBlockEventForm.vue', () => {
+  setupDayjs();
+  let wrapper: Wrapper<EthBlockEventForm>;
+
+  const groupHeader: EthBlockEvent = {
+    identifier: 11336,
+    entryType: HistoryEventEntryType.ETH_BLOCK_EVENT,
+    eventIdentifier: 'BP1_444',
+    sequenceIndex: 0,
+    timestamp: 1697442021000,
+    location: 'ethereum',
+    asset: 'ETH',
+    balance: {
+      amount: bigNumberify('100'),
+      usdValue: bigNumberify('2000')
+    },
+    eventType: 'staking',
+    eventSubtype: 'mev reward',
+    locationLabel: '0x106B62Fdd27B748CF2Da3BacAB91a2CaBaeE6dCa',
+    notes:
+      'Validator 12 produced block 444 with 100 ETH going to 0x106B62Fdd27B748CF2Da3BacAB91a2CaBaeE6dCa as the mev reward',
+    validatorIndex: 122,
+    blockNumber: 444
+  };
+
+  const createWrapper = (options: ThisTypedMountOptions<any> = {}) => {
+    const vuetify = new Vuetify();
+    const pinia = createPinia();
+    setActivePinia(pinia);
+    return mount(EthBlockEventForm, {
+      pinia,
+      vuetify,
+      stubs: {
+        VAutocomplete: {
```

Comment:
Is this stub the same across different specs? if so do you think we can extract it somewhere?